specs.md

```diff
@@ -59,7 +59,7 @@ FSM-5 | APPS open/short circuit | If there is an open/short circuit in either en
 FSM-6 | APPS disagreement | - When the primary and secondary APPS positions disagree by more then 10%, and the disagreement persists for more than 100ms, the FSM must throw a motor shutdown fault and report 0% mapped pedal percentage. <br/> - The FSM must clear the motor shutdown fault after the APPS positions agree within 10%, and the agreement persists for more than 1s. | T.4.2.3, T.4.2.4, T.4.2.5
 FSM-7 | APPS/brake pedal plausibility check | - When the APPS senses brake actuation and more than 25% mapped pedal percentage simultaneously, the FSM must throw a motor shutdown fault and report 0% mapped pedal percentage. <br/> - The FSM must clear the motor shutdown fault after the APPS senses less than 5% mapped pedal percentage, regardless of the brake state. | EV.3.4.1, EV.3.4.2
 FSM-8 | Steering angle reporting | - The FSM must report the steering angle in degrees over CAN at 1kHz, where 0 degrees represents straight wheels and a clockwise turn of the steering wheel corresponds to an increase in steering angle. <br/> - The FSM must send a non-critical fault when the steering angle is beyond the max turning radius of the steering wheel.
-FSM-9 | Wheel speed reporting | - The FSM must report the two front wheel speeds in km/h over CAN at 1kHz. <br/> - The FSM must send a non-critical fault when either front wheel speed is below -10km/h or above 150km/h.
+FSM-9 | Wheel speed reporting | - The FSM must report the two front wheel speeds in km/h over CAN at 1kHz. <br/> - The FSM must send a non-critical fault when either front wheel speed is inactive or above 150km/h.
```

Comment:
Inactive is not well defined, it reads like 0km/hr

Reply:
fixed

Reply:
I thought about it and got rid of inactive. I kept the upper threshold doe.

Reply:
Yeah i'm not so sure about the negative wheel speed, maybe ask @kirons96 what he had in mind? Maybe there's a way to measure negative speed and i'm just unaware?

Reply:
I talked to Prom and we couldn't figure a way out. best way to do this is to move this functionality to the DCM as the inverter can read negative speeds. please add a -10km/h check on the DCM